config/reasoner/op2dl/main/mapping.n3

```diff
@@ -33,13 +33,14 @@ PREFIX xsd: <http://www.w3.org/2001/XMLSchema#>
   ?s a besluit:Bestuurseenheid
 }.
 
-{
-  ?s a ere:RepresentatiefOrgaan
-}=> {
-  ?s
-    a besluit:Bestuurseenheid;
-    besluit:classificatie <http://data.vlaanderen.be/id/concept/BestuurseenheidClassificatieCode/36372fad-0358-499c-a4e3-f412d2eae213>.
-}.
+# this rule should no longer be needed as OP has a classification for RO now
+# {
+#  ?s a ere:RepresentatiefOrgaan
+# }=> {
+#  ?s
+#    a besluit:Bestuurseenheid;
```

Comment:
I think that we still need to add the `besluit:Bestuurseenheid` type because ACMIDM needs it for the log in ? Not 100% sure, but worth checking